mqtt/mqtt-bridge/config/default.json

```diff
@@ -0,0 +1,14 @@
+{
+    "subscriptions": [
+        { 
+            "pattern": "temp/#",
+            "remote": "floor/kitchen/"
```

Comment:
this looks like test data, so maybe we should move it to test instead of default.json config?

Reply:
moved to test